vobject/behavior.py

```diff
@@ -81,22 +82,52 @@ def validate(cls, obj, raiseException=False, complainUnrecognized=False):
             return cls.lineValidate(obj, raiseException, complainUnrecognized)
         elif isinstance(obj, base.Component):
             count = {}
+            language_count = {} #example: {'SUMMARY': {'en': 1, 'ja': 1} , 'DESCRIPTION': {None: 1}}
             for child in obj.getChildren():
                 if not child.validate(raiseException, complainUnrecognized):
                     return False
                 name = child.name.upper()
                 count[name] = count.get(name, 0) + 1
+
+                if isinstance(child, base.ContentLine) and name in properties_allow_localization:
+                    # for some properties, we count separately with respect to language
+                    lang = child.params.get('LANGUAGE', [])
+                    if len(lang) == 0:
+                        lang = None
+                    elif len(lang) == 1:
+                        lang = lang[0].lower()
+                    elif len(lang) > 1:
+                        if raiseException:
+                            m = "Multiple language parameters specified for property {1}"
+                            raise base.ValidateError(m.format(name))
+                        return False
+                    if name in language_count:
+                        language_count[name][lang] = language_count[name].get(lang, 0) + 1
+                    else:
+                        language_count[name] = {lang: 1}
```

Comment:
Either only line 105 and no if-statement at all, or change line 105 to
```
language_count[name][lang] += 1
```
since you ensured that it exists in line 104